lib/drivers/cryptoid.js

```diff
@@ -74,7 +74,7 @@ class CryptoID extends Driver {
 
   /**
    * @augments Driver.getSupply
-   * @param {Coin} modifiers {@link Coin}
+   * @param {Coin} blockchain {@link Coin}
```

Comment:
This is not really correct. To let the getSupply function work you should pass along a coin which contains the blockchain. The function destructures the coin immediately, but it would still be a coin :)
```suggestion
   * @param {Coin} coin {@link Coin}
```

Reply:
@nickpater make sense. tnx